waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -931,7 +946,8 @@
                   ;; but store the full service-id as an annotation.
                   ;; https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set
                   ;; https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#syntax-and-character-set
-                  :annotations {:waiter/service-id service-id}
+                  :annotations {:waiter/revision-timestamp revision-timestamp
```

Comment:
You injected this new binding between the comment about storing the service-id as an annotation and the service-id binding.

Reply:
Moved the comment closer to the `service-id` annotation.